sympy/codegen/tests/test_cnodes.py

```diff
@@ -26,14 +27,19 @@ def test_goto_Label():
     g = goto(s)
     assert g.func(*g.args) == g
     assert g != goto('foobar')
-    assert ccode(g) == 'goto early_exit'
-
-    l = Label(s)
-    assert l.is_Atom
-    assert ccode(l) == 'early_exit:'
-    assert g.label == l
-    assert l == Label(s)
-    assert l != Label('foobar')
+    assert ccode(g) == 'goto early_exit;'
+
+    l1 = Label(s)
+    assert ccode(l1) == 'early_exit:'
+    assert l1 == Label(s)
```

Comment:
Is this really useful ?
`Label('early_exit')` might be better

Reply:
Yeah! indeed... :sweat_smile:

Reply:
It's the same but it is currently testing l1 == l1.
At least Label('early_exit') will test, it has the right label.